### PR TITLE
Dependency Updates

### PR DESCRIPTION
**Pull request by speedy1812**
This PR updates both the Ruby and Node dependencies. Updating from Gulp 3.x to 4.x also necessitated some changes to the Gulpfile. 

I mostly referenced the PR where I updated my blog to Gulp 4.x: https://github.com/speedy1812/daysinukraine.com/pull/5

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1,2 +1,2 @@
-ruby 2.6.5
-nodejs 10.15.3
+ruby 2.6.6
+nodejs 12.18.2
```
Comment (speedy1812) [inline .tool-versions:2]: There are newer versions of Ruby and Node, but this is where my blog is at, and I know these work. Maybe once I update my blog, I'll come back and update them. 🙂

**Comment by speedy1812** [comment]
Also, I haven't actually run a deploy on this branch yet. The `npm run production` task seems to be running fine, but it would be good to at least deploy to staging to make sure everything is working.

**Comment by joshukraine** [comment]
@speedy1812 Awesome, thanks for doing this! I'll pull it down and have a look...

**Review by joshukraine — APPROVED** [review]
Looks good! 😃

**Pull request merged**